lib/msal-browser/src/app/ClientApplication.ts

```diff
@@ -502,11 +508,9 @@ export abstract class ClientApplication {
      * MSAL return's a cached token when available
      * Or it send's a request to the STS to obtain a new token using a refresh token.
      *
-     * @param {@link (SilentRequest:type)}
      *
-     * To renew idToken, please pass clientId as the only scope in the Authentication Parameters
+     * @param request
```

Comment:
Why are we losing comments here?

Reply:
This comment is no longer true in MSAL Browser.